locales/es.js

```diff
@@ -0,0 +1,218 @@
+	"TryChooser syntax and pushing to try": "Sintaxis de TryChooser y enviando a try",
+	"Running buildbot tests locally": "Ejecutando pruebas de buildbot localmente",
+	"Introduction to the XPCShell and XPCShell tests": "Introducción a XPCShell y las pruebas XPCShell",
+	"Basics of creating an XPCShell test": "Lo básico de crear una prueba XPCShell",
+	"Creating a basic mochitest": "Creando un mochitest básico",
+	"Introduction to reftests": "Introducción a los reftests",
+	"Basics of creating a reftest": "Lo básico de crear un reftest",
+	"Creating a basic browser chrome test": "Creando una prueba básica del chrome del navegador",
+	"Attaching a patch to Bugzilla and getting your first review": "Adjuntando un parche en Bugzilla y obteniendo tu primera revisión",
+	"Review results, integration branches, and landing your patch": "Resultados de revisión, ramas de integración y aplicando tu parche",
+	"Debugging your local build with Visual Studio": "Depurando tu compilación local con Visual Studio",
+	"Introduction to different debugger windows in Visual Studio": "Introducción a diferentes ventanas de depuración en Visual Studio",
+	"More advanced debugging tricks with Visual Studio": "Trucos de depuración más avanzados en Visual Studio",
+	"Debugging your local build in OS X with Xcode": "Depurando tu compilación local en OS X con Xcode",
+	"Debugging your local builds in Linux with gdb and ddd": "Depurando tus compilaciones locales en Linux con gdb y ddd",
+	"Creating C++ XPCOM components": "Creando componentes XPCOM en C++",
+	"Initialize data": "Inicializando datos",
+	"Reads data from videos.json and puts it into the redis DB.": "Lee datos de videos.json y los coloca en la base de datos redis.",
+	"Also re-initializes all controllers": "También re-inicializa todos los controladores",
+	"Server Statistics": "Estadísticas del servidor",
+	"Last server instance reset: ": "Último reinicio de la instancia del servidor: ",
+	" ago": " atrás",
+	"User Statistics": "Estadísticas del usuario",
+	"These statistics are only tracked for logged on users": "Estas estadísticas son solo realizadas para los usuarios conectados",
+	"User count: ": "Contador de usuarios: ",
+	"Lessons Completed: ": "Lecciones completadas: ",
+	"Completed per user: ": "Completadas por usuario: ",
+	"Linked Bugzilla accounts: ": "Cuentas de Bugzilla enlazadas: ",
+	"Display names filled: ": "Nombres para mostrar rellenados: ",
+	"Websites filled: ": "Sitios web rellenados: ",
+	"Days since last joined: ": "Días desde la última conexión: ",
+	"Lessons completed by user": "Lecciones completadas por usuario",
+	" completed": " completadas",
+	"Lessons completed by lesson": "Lecciones completas por lección",
+	"Perform the following tasks in JavaScript:": "Realizar las siguientes tareas en JavaScript:",
+	"Not yet completed": "Aún no completado",
+	"So far so good.": "Hasta ahora, todo bien.",
+	"Do not have any syntax errors.": "No cometas errores de sintaxis.",
+<<<<<<< HEAD
```

Comment:
Basically just these lines need to be deleted:
<<<<<<< HEAD on line 207
======= on line 212
and 
>>>>>>> bf08ee362e6380fada6aa49e7de33b5d44e99012
on line 218

After you remove them you can git add -u 
Then commit --amend